packaging_automation/dbconfig.py

```diff
@@ -27,3 +29,20 @@ def db_session(db_params: DbParams, is_test: bool, create_db_objects: bool = Tru
 
 
 Base = declarative_base()
+
+
+class RequestType(Enum):
+    docker_pull = 1
+    github_clone = 2
+    package_cloud_list_package = 3
+    package_cloud_detail_query = 4
+    homebrew_download = 5
+
+
+class RequestLog(Base):
+    __tablename__ = "request_log"
+    id = Column(INTEGER, primary_key=True, autoincrement=True)
+    request_time = Column(TIMESTAMP, nullable=False)
+    request_type = Column(String, nullable=False)
```

Comment:
Shouldn't this be `RequestType` instead of `String`?

Reply:
Don't know whether SqlAlchemy has Enum data dype. Thanks to you actually it has one :) Fixed it